src/averageAges.js

```diff
@@ -20,6 +20,13 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const agesOfMen = people
+    .filter(({ sex, died }) => sex === 'm' && isDiedInCentury(died, century))
+    .map(person => person.died - person.born);
+  const totalAges = agesOfMen.reduce((a, b) => a + b, 0);
+
+  return (totalAges / agesOfMen.length) || 0;
```

Comment:
You use `reduce` and division by length in every function. Maybe we should move this logic to a separate helper?